app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
```

Comment:
According to stack overflow "What that warning is saying is that, according to rubocop's standard, you have too many assignments, branches, and conditionals (ABCs) in that method. "

That said, it looks fine to me. We haven't learned a cleaner way to do that, and you need to keep the params in the controller.